src/Avalonia.Styling/StyledElement.cs

```diff
@@ -470,12 +446,13 @@ void ISetLogicalParent.SetParent(ILogical parent)
                     var e = new LogicalTreeAttachmentEventArgs(newRoot, this, parent);
                     OnAttachedToLogicalTreeCore(e);
                 }
-
+#nullable disable
                 RaisePropertyChanged(
                     ParentProperty,
                     new Optional<IStyledElement>(old),
                     new BindingValue<IStyledElement>(Parent),
                     BindingPriority.LocalValue);
+#nullable enable
```

Comment:
`restore` might better actually.

Reply:
`restore` restores the setting to the project default afaiu.

Reply:
Right, we are still enabling it per file.